homeassistant/components/glances/coordinator.py

```diff
@@ -45,15 +45,13 @@ async def _async_update_data(self) -> dict[str, Any]:
         except exceptions.GlancesApiError as err:
             raise UpdateFailed from err
         # Update computed values
-        uptime: datetime | None = self.data["computed"]["uptime"] if self.data else None
+        uptime: datetime | None = None
```

Comment:
This could have just been left as it was as with below we know if there is `self.data` there is also `self.data["computed"]["uptime"]` (even if it might be `None`)

Reply:
If we initialize uptime here to the previous value, it never gets set to None if uptime is not available in the new data structure.
So uptime keeps the stale value indefinitely which made the new unit test fail.
It's not a big problem as uptime cannot be disabled in Glances (contrary to other modules like fs or network which triggered the need for this PR), so I can revert if needed.

Reply:
I will come back on this one shortly @wittypluck 
Have been occupied elsewhere unfortunately. Have not missed it